spring-rabbit/src/test/java/org/springframework/amqp/rabbit/core/RabbitManagementTemplateTests.java

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.amqp.rabbit.core;
+
+import static org.hamcrest.Matchers.anyOf;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import org.junit.After;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import org.springframework.amqp.core.Binding;
+import org.springframework.amqp.core.BindingBuilder;
+import org.springframework.amqp.core.DirectExchange;
+import org.springframework.amqp.core.Exchange;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.rabbit.connection.CachingConnectionFactory;
+import org.springframework.amqp.rabbit.support.Policy;
+import org.springframework.amqp.rabbit.test.BrokerRunning;
+
+/**
+ * @author Gary Russell
+ * @since 1.5
+ *
+ */
+public class RabbitManagementTemplateTests {
+
+	private final CachingConnectionFactory connectionFactory = new CachingConnectionFactory("localhost");
+
+	private final RabbitManagementTemplate template = new RabbitManagementTemplate();
+
+	@ClassRule
+	public static BrokerRunning brokerRunning = BrokerRunning.isRunning();
```

Comment:
I think this rule isn't enough for this test, because `management-plugin` must be enabled on the Broker.
At least on Windows I have to switch on it manually, as well as MQTT and STOMP.